src/services/query.js

```diff
@@ -357,9 +357,22 @@ ngeo.Query.prototype.issue = function(map, object) {
 ngeo.Query.prototype.issueIdentifyFeaturesRequests_ = function(map, coordinate) {
   var sources = this.getQueryableSources_(map, false);
 
-  this.doGetFeatureInfoRequests_(sources.wms, coordinate, map);
-  this.doGetFeatureRequestsWithCoordinate_(sources.wfs, coordinate, map);
-  this.updatePendingState_();
+  // Launch requests only if there are queryable sources
+  var doneRequest = false;
+
+  if (Object.keys(/** @type {!Object} */ (sources.wms)).length) {
+    this.doGetFeatureInfoRequests_(sources.wms, coordinate, map);
+    doneRequest = true;
+  }
+
+  if (Object.keys(/** @type {!Object} */ (sources.wfs)).length) {
+    this.doGetFeatureRequestsWithCoordinate_(sources.wfs, coordinate, map);
+    doneRequest = true;
+  }
+
+  if (doneRequest) {
+    this.updatePendingState_();
```

Comment:
Okay, I understand what's going on, now.  Originally, the code only supported WMS sources.  Once one was found to be "queryable", it was marked "pending" and a request was issued.  When WFS was implemented, the code that managed the getting of the queryable sources was moved to a method: `getQueryableSources_`.  That method still has the remains of what was done before, i.e. it marks found sources as `pending`.

In our case, we have only one source that doesn't support WFS.  So when obtaining it, it's marked as `pending` but in the end there's no request sent... but the `updatePendingState` doesn't behave correctly thereafter and we have the spinner being shown.

I think we should move the pending state of sources away from the  `getQueryableSources_` method.  I'll see what I can do.